fecgov/openFEC
# Update js-yaml package

## Summary (required)

Resolves #3888

- Update `js-yaml` JS package to address vulnerabilities
- Run `npm audit fix`
- Add `babel-preset-stage-2` to address build errors after these changes

## How to test the changes locally

```
npm install -g swagger-tools
npm install
npm run build
```
Run the API locally, test out the swagger interactions. Check out @jason-upchurch's recent PR for good instructions https://github.com/fecgov/openFEC/pull/3892

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Swagger API docs



## Related PRs
Recent JS package security update

branch | PR
------ | ------
`feature/3887-fix-high-security-vulnerability-lodash` | https://github.com/fecgov/openFEC/pull/3892
